README.md

```diff
@@ -12,6 +12,75 @@ The UCRM Plugin SDK can be installed with [Composer](https://getcomposer.org/).
 composer require ubnt/ucrm-plugin-sdk
 ```
 
+## Usage
```

Comment:
show this example below "Available classes"?

Reply:
sure, why not